src/pages/remote-services/Index.jsx

```diff
@@ -104,7 +111,7 @@ const RemoteServices = () => {
         {noOfServices > 0 && (
           <React.Fragment>
             <h3 style={{ display: "flex", justifyContent: "space-between" }}>Remote Services <Button onClick={() => setModalVisible(true)} type="primary">Add</Button></h3>
-            <Table columns={tableColumns} dataSource={servicesTableData} />
+            <Table onRowClick={handleViewClick} columns={tableColumns} dataSource={servicesTableData} onRow={(record) => { return { onClick: event => { handleViewClick(record.name) } } }} />
```

Comment:
Why are you using both `onRowClick` and `onRow`. Use any one of them which is working. And make sure you change this at both the places in the app